pkg/apis/config/v1beta1/types.go

```diff
@@ -60,3 +60,13 @@ type NodeResourcesAllocatableArgs struct {
 	// Whether to prioritize nodes with least or most allocatable resources.
 	Mode *ModeType `json:"mode,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CapacitySchedulingArgs defines the scheduling parameters for CapacityScheduling plugin.
+type CapacitySchedulingArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// KubeConfigPath is the path of kubeconfig.
+	KubeConfigPath string `json:"kubeconfigpath,omitempty"`
```

Comment:
nit: make it a *string in versioned pkg, so we can easily set a default value if it's nil; if it's a concrete string, we cannot tell if the zero value ("") is set by users.

Reply:
In general, we can set the default parameters `default.go` for all plugins: _CoScheduling, NodeResouresAllocatable, CapacityScheduling_.

Something like
```
func (csa *CapacitySchedulingArgs) SetDefaults() {
   defaultPath := "/var/lib/kubernetes/kubeconfig"
    if  csa.KubeConfigPath == nil {
       csa.KubeConfigPath = &defaultPath
    }
   ...
```

We can then validate the parameters in `validation.go`. 

```
// Validate checks and verifies the configuration parameters.
func (conf *CapacitySchedulingArgs) Validate() Error
     ...
}
```

Reply:
line 55: `// An example resource set might includes "cpu" (millicores) and "memory" (bytes)`
**includes** -> **include**

Reply:
> line 55: // An example resource set might includes "cpu" (millicores) and "memory" (bytes)
---
thanks. I address it together in this pr.

> In general, we can set the default parameters default.go for all plugins: CoScheduling, NodeResouresAllocatable, CapacityScheduling.

Done 

@yuanchen8911